MAUI/DataForm/Validation.md

```diff
@@ -15,6 +15,177 @@ The data form validates the data and user input to update the correct value in t
 
 The supported built in validations are as follows:
 
+#### IDataErrorInfo
+You can validate the data by implementing the `IDataErrorInfo` interface in the data object class.
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : IDataErrorInfo, INotifyPropertyChanged
+{
+    private int employeeID;
+    private string name;
+    private string title;       
+
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    public EmployeeInfo()
+    {
+
+    }
+
+    public int EmployeeID
+    {
+        get { return this.employeeID; }
+        set
+        {
+            this.employeeID = value;
+            this.OnPropertyChanged("EmployeeID");        
+        }
+    }
+    public string Name
+    {
+        get { return this.name; }
+        set
+        {
+            this.name = value;
+            this.OnPropertyChanged("Name");
+        }
+    }
+
+
+    public string Title
+    {
+        get { return this.title; }
+        set
+        {
+            this.title = value;
+            this.OnPropertyChanged("Title");
+        }
+    }
+
+    [Display(AutoGenerateField = false)]
+    public string Error
+    {
+        get
+        {
+            return string.Empty;
+        }
+    }
+
+    public string this[string columnName]
+    {
+        get
+        {
+            if (!columnName.Equals("Title"))
+                return string.Empty;
+
+            if (this.Title.Contains("Marketing"))
+                return "Marketing is not allowed";
+
+            return string.Empty;
+        }
+    }
+
+    private void OnPropertyChanged(string propertyName)
+    {
+        if (PropertyChanged != null)
+            PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+
+#### INotifyDataErrorInfo
+
+You can validate the data by implementing the `INotifyDataErrorInfo` interface in the data object class. This interface has three members,
+
+* HasErrors property
+* GetErrors method
+* ErrorsChanged event
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : INotifyDataErrorInfo, INotifyPropertyChanged
```

Comment:
Check is this required INotifyPropertyChanged

Reply:
INotifyPropertyChanged removed